src/main/java/com/google/cloud/tools/gradle/appengine/standard/DevAppServerStartTask.java

```diff
@@ -43,13 +45,22 @@ public void setCloudSdkBuilderFactory(CloudSdkBuilderFactory cloudSdkBuilderFact
   @TaskAction
   public void startAction() throws AppEngineException, IOException {
 
+    // Add a listener to write to a file for non-blocking starts, this really only works
+    // when the gradle daemon is running (which is default for newer versions of gradle)
+    File logFile = File.createTempFile("dev_appserver", ".out", getTemporaryDir());
```

Comment:
The assertion is to check only one log file is created. I guess I could add another test to make sure 2 log files are created for the same project. Projects start in a pristine state in this test (no build dir)